drivers/hmis/lib/form_data/default/fragments/signature.json

```diff
@@ -0,0 +1,100 @@
+{
+  "type": "GROUP",
+  "variant": "SIGNATURE",
+  "link_id": "signature_group",
+  "text": "eSignature",
+  "subtitle": "The following signatures are required on this document.",
+  "item": [
+    {
+      "type": "GROUP",
+      "text": "Head of Household",
+      "link_id": "head_of_household_signature_group",
+      "item": [
+        {
+          "type": "STRING",
+          "link_id": "head_of_household_signature",
+          "text": "Full Name",
+          "required": true,
+          "mapping": {
+            "custom_field_key": "head_of_household_signature"
+          }
+        },
+        {
+          "type": "DATE",
+          "link_id": "head_of_household_signature_date",
+          "text": "Today's Date",
+          "required": true,
+          "enable_when": [
```

Comment:
@ttoomey when we paired on Friday we had discussed this method of requirement validation, where some signatures are required and others are optional, and the date is always required if the signature is non-null.

I think I set it up the way we had discussed, and I'm wondering now if this is really the desired behavior. One thing I noticed about it is that this way the "Required" label gets rendered next to both the signature and the date -- whereas in Ash's designs, the label is rendered next to the heading ("Head of Household" or "Case Manager") which makes a little more sense from a ux perspective, I think.

Reply:
It's a good point. I could see an argument either way although TBH I would lean more towards keeping the required label next to the field (as you have it). Repeating the text is seems more clear and it is consistent with the treatment of other required fields in the app